### PR TITLE
Feature/positions model

### PR DESCRIPTION
**Pull request by diegosaul402**
Add position model along with its test, and factory.

### CODE REVIEW

File: spec/factories/department.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :department, class: Feedbook::Department do
+    name 'Department'
+  end
+end
```
Comment (gerardolopezduenas) [inline spec/factories/department.rb]: Github complains that there is no newline at the end of the file, maybe your editor is not using LF notation.
What do you think about to add a newline at end of the file?

File: spec/factories/position.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :position, class: Feedbook::Position do
+    name 'CEO'
+    job_description 'Rule them All'
+    association :department, factory: :department
+  end
+end
```
Comment (gerardolopezduenas) [inline spec/factories/position.rb]: Github complains that there is no newline at the end of the file, maybe your editor is not using LF notation.
What do you think about to add a newline at end of the file?

File: spec/models/feedbook/position_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Feedbook::Position, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:job_description) }
+  it { should belong_to(:department) }
+  it 'should create a position by factory method' do
```
Comment (JuanCrg90) [inline spec/models/feedbook/position_spec.rb]: In tests like this is more common to use a text like:

`creates a position by factory method` should is unnecessary.

**Comment by diegosaul402** [comment]
Thank you, please review again

**Pull request merged**